### PR TITLE
Add mock for Share API

### PR DESCRIPTION
**Pull request by m-vdb**
Hi there,

This is the mock for the [Share API](https://facebook.github.io/react-native/docs/share.html).

Cheers!

### CODE REVIEW

File: src/api/Share.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+class Share {
+
+  static share(content, options) {
+    return Promise.resolve('sharedAction');
+  }
+
+  static get sharedAction() { return 'sharedAction'; }
```
Comment (RealOrangeOne) [inline src/api/Share.js]: Could this drop to multiple lines, or use fat-arrow syntax.

File: src/api/Share.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  static get dismissedAction() { return 'dismissedAction'; }
```
Comment (RealOrangeOne) [inline src/api/Share.js]: Could this drop to multiple lines, or use fat-arrow syntax.

**Review by RealOrangeOne — CHANGES_REQUESTED** [review]
Great work!

Couple comments (both the same).

Also, please fix linter. See travis logs for exact errors

**Comment by m-vdb** [comment]
thanks @RealOrangeOne it should be good with: f19363e

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**